<launch>

    <!-- Launch openpose_ros_control node -->
    <node pkg="openpose_ros_control" type="openpose_ros_control_node" name="openpose_ros_control_node" output="screen">
        <rosparam command="load" file="$(find openpose_ros_control)/config/openpose_ros_control.yaml"/>
    </node>

</launch>